### DISCUSSION TITLE
多源数据目录功能问题

### DISCUSSION THREAD

**Discussion by gaojunjie03** [apache/doris#14391] [A - General / Q&A]
doris版本是主分支版本，commit id：24b51b9

使用这个多源数据目录去创建hive的catalog，里面有hive表和hudi表，查询hive表的时候返回空数据，实际上里面是有数据的（1条）。而查询hudi的表则报错：
Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=181) errCode = 2, detailMessage = failed to find default value expr for slot: id_

然后我用doris on hive的方式去查询hive表，也是报错：
Failed to execute sql: java.sql.SQLException: (conn=184) errCode = 2, detailMessage = file: /warehouse/tablespace/managed/hive/demo.db/student/delta_0000001_0000001_0000/_orc_acid_version error:Failed to create orc file reader

hive版本是：3.0.0.3.1
hdfs版本是：3.1.1.3.1
hudi版本是：0.12.1

**Comment by tiger-hcx** [comment]
CREATE CATALOG hive7 PROPERTIES (
    'type'='hms',
    'hive.metastore.uris' = 'thrift://ip:9083'
);
我已成功在doris中通过连接hive元数据读取hudi的数据
